### PR TITLE
feat: add radians and degrees functions

### PR DESCRIPTION
**Pull request by richtia**
PR to add radians and degrees functions

### CODE REVIEW

**Review by EpsilonPrime — DISMISSED** [review]
Is it worth putting these functions in a new file (extensions/geometric.yaml perhaps) to keep the file size down?  Two small functions aren't going to make much of a difference though.

**Comment by richtia** [comment]
> Is it worth putting these functions in a new file (extensions/geometric.yaml perhaps) to keep the file size down? Two small functions aren't going to make much of a difference though.

I thought these made more sense going next to the trig functions.  I was planning on adding mostly geospatial functions from postgis to the geometric yaml.

**Comment by EpsilonPrime** [comment]
Works for me.

File: extensions/functions_arithmetic.yaml
Comment (gforsyth) [inline extensions/functions_arithmetic.yaml:655]: Most extension functions avoid this kind of implicit upcasting -- is there a particular reason we want this to return `fp64` for a `fp32` input?

**Review by gforsyth — COMMENTED** [review]
Small question about the output types -- and should we also add in `rounding` behavior for the floating point args?

File: extensions/functions_arithmetic.yaml
Comment (richtia) [reply extensions/functions_arithmetic.yaml:655]: oh yeah...we shouldn't be implicit upcasting here.  I'll update it!  I'll also add the rounding options.

**Review by gforsyth — APPROVED** [review]
thanks @richtia!  this looks good to me!

@EpsilonPrime -- does this need PMC approval since it's not changing existing functionality?

**Comment by EpsilonPrime** [comment]
> thanks @richtia! this looks good to me!
> 
> @EpsilonPrime -- does this need PMC approval since it's not changing existing functionality?

Technically probably since it adds functionality.  But given that this isn't controversial I don't think anyone would mind.

**Comment by EpsilonPrime** [comment]
> > thanks @richtia! this looks good to me!
> > @EpsilonPrime -- does this need PMC approval since it's not changing existing functionality?
> 
> Technically probably since it adds functionality. But given that this isn't controversial I don't think anyone would mind.

Hmm, but a reread of substrait.io/governance does explicitly state that addition of a non-breaking function merely requires one committer to approve.  So we should be good to merge this one.

**Pull request merged**
